src/api/spec/features/beta/webui/decisions_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'browser_helper'
+
+RSpec.describe 'Decisions', :js, :vcr do
+  before do
+    Flipper.enable(:content_moderation)
+  end
+
+  let(:reporter) { create(:confirmed_user, login: 'foo') }
+  let(:moderator) { create(:moderator, login: 'baz') }
+
+  def fill_decisions_modal(reportable)
+    within("#reports-modal-#{reportable.class.to_s.downcase}-#{reportable.id}") do
+      fill_in id: 'decision_reason', with: 'Reason for reporting is correct.'
+      select('favor', from: 'decision[kind]')
+      click_button('Submit')
+    end
+  end
+
+  describe 'project' do
+    let(:project) { create(:project, name: 'against_the_rules') }
+    let!(:report_for_project) { create(:report, reportable: project, reason: 'This project does not follow the rules!', user: reporter) }
+
+    before do
+      login(moderator)
+      visit project_show_path(project)
+    end
+
+    it 'creates the decision for a moderator' do
+      expect(page).to have_text('This project has 1 report .')
+      click_link('1 report')
+      fill_decisions_modal(project)
+
+      expect(page).to have_text('Decision created successfully')
+      expect(Decision.count).to eq(1)
```

Comment:
We should be using the [change rspec matcher](https://www.rubydoc.info/gems/rspec-expectations/RSpec/Matchers:change) for this kind test expectations

Reply:
I agree, we already do in some specs. I didn't think about that during the review.